tests/vppinputoutput.h

```diff
@@ -252,13 +252,13 @@ class VppOutput
 {
 public:
     static SharedPtr<VppOutput>
-        create(const char* outputFileName);
+        create(const char* outputFileName, uint32_t fourcc = 0, int width = 0, int height = 0);
     bool getFormat(uint32_t& fourcc, int& width, int& height);
     virtual bool output(const SharedPtr<VideoFrame>& frame) = 0;
     VppOutput();
     virtual ~VppOutput(){}
 protected:
-    virtual bool init(const char* outputFileName) = 0;
+    virtual bool init(const char* outputFileName, uint32_t fourcc = 0, int width = 0, int height = 0) = 0;
```

Comment:
do not need = 0 since "create(const char* inputFileName, uint32_t fourcc = 0, int width = 0, int height = 0)" can make sure this